src/matcher_combinators/matchers.clj

```diff
@@ -40,18 +48,10 @@
   ([select-fn expected]
    (core/->SelectingInAnyOrder select-fn expected)))
 
-(defn sublist
+(defn prefix-seq
   "Matcher that will match when provided a (ordered) prefix of the `expected`
   list.
 
-  Similar to Midje's `(contains expected)`"
-  [expected]
-  (core/->SubSeq expected))
-
-(defn subset
```

Comment:
matching an unordered subset of elements in a sequence used to be `subset`. It is now called `embeds`, does that seem like a good name for it?